CONTRIBUTING.md

```diff
@@ -30,11 +30,15 @@ The sections below outline the steps in each case.
 
 1. (**important**) announce your plan to the rest of the community _before you start working_. This announcement should be in the form of a (new) issue;
 1. (**important**) wait until some kind of consensus is reached about your idea being a good idea;
-1. if needed, fork the repository to your own Github profile and create your own feature branch off of the latest master commit. While working on your feature branch, make sure to stay up to date with the master branch by pulling in changes, possibly from the 'upstream' repository (follow the instructions [here](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and [here](https://help.github.com/articles/syncing-a-fork/));
+1. if needed, fork the repository to your own Github profile and create your own feature branch off of the latest master
+   commit. While working on your feature branch, make sure to stay up to date with the master branch by pulling in
```

Comment:
I think this one shouldn't be on a new line

Reply:
This was auto formatted to stay within maximum line length. It is not parsed as a new line [when displayed on github](https://github.com/neurogym/neurogym/blob/dev_readme_dbodor/CONTRIBUTING.md#you-want-to-make-some-kind-of-change-to-the-code-base).